### ISSUE TITLE
ListenAddress don't taken into account

### ISSUE THREAD

**Issue by belfo** [oracle/weblogic-kubernetes-operator#2459]
Hello,

I updated the model-in-image example to set a ListenAddress for the admin server on 0.0.0.0 (to be able to access the console from outside using a port forward)
But the generated domain still have the : 
<listen-address>sample-domain1-admin-server</listen-address>

Is this the expected behaviour?

Regards

**Comment by rjeberhard** [comment]
Yes, this is the expected behavior. Information about listen addresses and some other information (such as node manager configurations) are overwritten with the in-cluster values as domains are deployed.

I don't quite understand the need to set the ListenAddress to 0.0.0.0 in order to access the console from outside with a port forward. Let me see if I can dig up a sample of using port forwarding.

**Comment by belfo** [comment]
Hello,

i get this error (on the port forward command)
E0715 15:59:30.499417 4070974 portforward.go:400] an error occurred forwarding 8080 -> 7001: error forwarding port 8001 to pod d0a9e46791541ebf798d93a9c4278fd15f7c4dbfc6bb506b57dba2c34ddc50fb, uid : exit status 1: 2021/07/15 13:59:30 socat[3493906] E connect(5, AF=2 **127.0.0.1**:7001, 16): Connection refused
And no connection arrive to the container who is listening only on the adress: sample-domain1-admin-server who resolve to 172.17.0.7

Webloigc logs print: <Jul 15, 2021 2:52:57,075 PM GMT> <Notice> <Server> <BEA-002613> <Channel "Default" is now listening on 172.17.0.7:7001 for protocols iiop, t3, ldap, snmp, http.> 

So that's what let me think the 0.0.0.0 is what i need in listen address. (as we use today running weblogic not in K8S)

**Comment by belfo** [comment]
@rjeberhard Did you find any example?

**Comment by tbarnes-us** [comment]
The operator documentation now outlines how to get port forward access to the console, with dependencies on Istio or, alternatively, upgrading to 3.3.2.

See [ > User Guide > Manage WebLogic domains > Access the domain > Use port forwarding](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/accessing-the-domain/port-forward/)
